<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorWindow</class>
 <widget class="QWidget" name="AdministratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="userListView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="excludeUserPushButton">
       <property name="text">
        <string>Исключить из чата</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Имя нового пользователя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newUserNameLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addUserPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Добавить пользователя</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
